### ISSUE TITLE
Attached files - Error 403

### ISSUE THREAD

**Issue by Sinetiqueta** [opensupports/opensupports#77]
I'm ableto attch files, but when I want to see, error 403 appears in my screen.
I think that this issue is related about permisions and is sometrhing that shoud be solved by my Server Company.
am I in the correct way?

regards

**Comment by ivandiazwm** [comment]
I don't think this has to do with server permissions, we manage file permission via a PHP script.
In order to download a file, you need to be either the author of the ticket/comment with attachment, or a staff member that has assigned the ticket's department.

Please let me know how are you trying to download the file.

**Comment by Sinetiqueta** [comment]
I was confused with Error 406, related with the "mod_security".
 
I'm going to check with your comments, and I will update you asap.

Thanks!

**Comment by Sinetiqueta** [comment]
Nop...

I'm able to upload as "client", but as Client or as Administrator, when I click on the file, apears ERROR 403:

![image](https://user-images.githubusercontent.com/31955294/30471858-8cae8f44-99fa-11e7-8c8a-4974e3a6e212.png)

regards!
